app/main.py

```diff
@@ -1,3 +1,60 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __add__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(other.km + self.km)
+        else:
```

Comment:
Remove redundant `else`, the code will not execute after return anyway. Apply this change to all relevant places